Source/Editor/Project/Project.cpp

```diff
@@ -498,13 +501,22 @@ void Project::InitializeDefaultProject()
 
 void Project::InitializeResourceCache()
 {
+    auto engine = GetSubsystem<Engine>();
     auto cache = GetSubsystem<ResourceCache>();
     cache->ReleaseAllResources(true);
     cache->RemoveAllResourceDirs();
     cache->AddResourceDir(dataPath_);
     cache->AddResourceDir(coreDataPath_);
     cache->AddResourceDir(cachePath_);
     cache->AddResourceDir(oldCacheState_.GetEditorData());
+
+    auto vfs = GetSubsystem<VirtualFileSystem>();
+    vfs->UnmountAll();
+    vfs->Mount(MakeShared<MountedDirectory>(context_, oldCacheState_.GetEditorData()));
+    vfs->Mount(MakeShared<MountedDirectory>(context_, coreDataPath_));
+    vfs->Mount(MakeShared<MountedDirectory>(context_, dataPath_));
+    vfs->Mount(MakeShared<MountedDirectory>(context_, cachePath_));
+    vfs->Mount(MakeShared<MountedDirectory>(context_, engine->GetAppPreferencesDir(), "conf"));
```

Comment:
If a small quality of life change would be for vfs to construct that object, and then we can just do `vfs->Mount(oldCacheState_.GetEditorData());`.

Reply:
Yep, I had it at some moment but rolled it back for uniformity. Since then I've simplified the Mounted directory construction so I can bring the method back